ext/drmgr/drmgr.c

```diff
@@ -208,6 +208,9 @@ typedef struct _per_thread_t {
     instr_t *last_instr;
     emulated_instr_t emulation_info;
     bool in_emulation_region;
+    /* This field stores the current insertion event instruction so that
+     * separate query API's don't need to take in the instruction.
```

Comment:
APIs? without the apostrophe

Reply:
Actually plurals of capital letter sequences are supposed to use apostrophes.  I guess the style preference varies: I think everyone agrees single letters do use apostrophes (`A's` means plural, not possessive); maybe there's some disagreement on multiple sequences with vs without periods.  OK I found a quote from the NY Times style guide https://afterdeadline.blogs.nytimes.com/2010/04/13/faqs-on-style/: with periods they use apostrophe; without they don't.  OK I will go with that.

Reply:
:D TIL apostrophes are used for plurals in some cases. Thanks.